modules/stanford_jse_dept/stanford_jse_dept.module

```diff
@@ -78,4 +78,39 @@ function stanford_jse_dept_css_alter(&$css) {
       break;
     default:
   }
-}
\ No newline at end of file
+
+  // Exclude admin pages
```

Comment:
From line 82 to line 114 it does not appear to be doing anything. Please remove or revise.